packages/inter-protocol/test/swingsetTests/psmUpgrade/bootstrap-psm-upgrade.js

```diff
@@ -13,18 +13,16 @@ import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
 import { E } from '@endo/eventual-send';
 import { Far } from '@endo/marshal';
 import { makePromiseKit } from '@endo/promise-kit';
-import { withAmountUtils } from '../../supports.js';
+import { scale6, withAmountUtils } from '../../supports.js';
```

Comment:
Just a comment: if you have `.units`, do you ever want to use `scale6()`?

Reply:
the one place I think we'll still need it: `assertChange`, because the brand doesn't change.

there are some other test utils that take bigints for amounts, like `mintRunPayment`. now that we have a more convenient way to make amounts we could change those to take branded amounts.